change/@fluentui-react-dialog-dc310d89-6297-481f-a1fa-2a25450bc15a.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "feat(react-dialog): 1st rule of ARIA for Dialog",
```

Comment:
I'll be addressing any Safari related issues on a separate PR, since some cases need investigation @layershifter, but in the worst case scenario of not getting native `<dialog>`to work out of of the box then I'll have to default to `div`and add `dialog` as an option